src/fetch/cow_rewards.py

```diff
@@ -3,36 +3,20 @@
 from web3 import Web3
 
 
-def map_reward(
-    amount: float,
-    risk_free: bool,
-    batch_contains_unsafe_liquidity: bool,
-) -> float:
+def map_reward(amount: float, risk_free: bool) -> float:
     """
     Converts orderbook rewards based on additional information of
     "risk_free" batches and (un)safe liquidity orders.
     - risk-free are batches contain only user and liquidity orders (i.e. no AMM interactions),
     - liquidity orders are further classified as being safe or unsafe;
```

Comment:
would be good to fix the comment as well

Reply:
Fixed.